client/extensions/wp-super-cache/miscellaneous.jsx

```diff
@@ -0,0 +1,190 @@
+									{
+										components: { em: <em /> }
+									}
+								) }
+							</span>
+						</FormToggle>
+
+						<FormToggle
+							checked={ !! cache_rebuild_files }
+							onChange={ handleToggle( 'cache_rebuild_files' ) }>
+							<span>
+								{ translate(
+									'Cache rebuild. Serve a supercache file to anonymous users while a new ' +
+									'file is being generated. {{em}}(Recommended){{/em}}',
+									{
+										components: { em: <em /> }
+									}
+								) }
+							</span>
+						</FormToggle>
+
+						<FormToggle
+							checked={ !! wp_supercache_304 }
+							disabled={ '1' === super_cache_enabled }
+							onChange={ handleToggle( 'wp_supercache_304' ) }>
+							<span>
+								{ translate(
+									'304 Not Modified browser caching. Indicate when a page has not been ' +
+									'modified since it was last requested. {{em}}(Recommended){{/em}}',
+									{
+										components: { em: <em /> }
+									}
+								) }
+							</span>
+							{ '1' === super_cache_enabled &&
+								<FormSettingExplanation>
+									{ translate(
+										'{{strong}}Warning! 304 browser caching is only supported when mod_rewrite caching ' +
+										'is not used.{{/strong}}',
+										{
+											components: { strong: <strong /> }
+										}
+									) }
+								</FormSettingExplanation>
+							}
+							{ '1' !== super_cache_enabled &&
+								<FormSettingExplanation>
+									{ translate(
+										'304 support is disabled by default because some hosts have had problems with the ' +
+										'headers used in the past.'
+									) }
+								</FormSettingExplanation>
+							}
+						</FormToggle>
+
+						<FormToggle
+							checked={ !! wp_cache_no_cache_for_get }
+							onChange={ handleToggle( 'wp_cache_no_cache_for_get' ) }>
+							<span>
+								{ translate( 'Don’t cache pages with GET parameters. (?x=y at the end of a url)' ) }
+							</span>
+						</FormToggle>
+
+						<FormToggle
+							checked={ !! wp_cache_make_known_anon }
+							onChange={ handleToggle( 'wp_cache_make_known_anon' ) }>
+							<span>
+								{ translate( 'Make known users anonymous so they’re served supercached static files.' ) }
+							</span>
+						</FormToggle>
+
+						<FormToggle
+							checked={ !! wp_cache_hello_world }
+							onChange={ handleToggle( 'wp_cache_hello_world' ) }>
+							<span>
+								{ translate( 'Proudly tell the world your server is {{fry}}Stephen Fry proof{{/fry}}! ' +
+									'(places a message in your blog’s footer)',
+									{
+										components: {
+											fry: (
+												<ExternalLink
+													icon={ true }
+													target="_blank"
+													href="https://twitter.com/#!/HibbsLupusTrust/statuses/136429993059291136"
+												/>
+											),
+										}
+									}
+							) }
+							</span>
+						</FormToggle>
+					</FormFieldset>
+				</form>
+			</Card>
+		</div>
+	);
+};
+
+const getFormSettings = settings => {
```

Comment:
I see this pattern in many of the files: a function we pass into `WrapSettingsForm` that returns a subset of settings fields. Was wondering if we should make this more declarative, by just passing the string array. 

Although passing in a function is more flexible.